### PR TITLE
Add embed.min.js and count.min.js for Isso

### PR DESCRIPTION
**Pull request by SnorlaxYum**
Pull request for issue: #9776 
Related issue(s): #9776 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/posativ/isso.git
 * Official website (optional, not the repository):https://posativ.org/isso/
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 1,718
   - Count of watchers: 78
   - Count of forks: 150
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [ ] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [ ] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
@SnorlaxYum there should be a package.json under the library directory

File: ajax/libs/isso/package.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "isso",
+  "filename": "embed.min.js" , "count.min.js",
```
Comment (PeterDaveHello) [inline ajax/libs/isso/package.json]: There can be only one filename to point to the main file.

**Comment by SnorlaxYum** [comment]
@PeterDaveHello Just changed that, is that okay?

**Comment by PeterDaveHello** [comment]
@SnorlaxYum just asking, do you commit the file on local or on GitHub web GUI?

**Comment by SnorlaxYum** [comment]
@PeterDaveHello on GitHub web GUI. What's wrong with that?

**Comment by PeterDaveHello** [comment]
@SnorlaxYum Thanks I got it, we'll squash the commits and rebase with master branch locally.

**Comment by SnorlaxYum** [comment]
Thanks! When it's merged into the master branch, notify me :smile:

**Comment by pvnr0082t** [comment]
@SnorlaxYum Where did you get the file? Would you please commit unminified also? Thank you.

**Comment by SnorlaxYum** [comment]
@pvnr0082t Just added auto-update. 
The js files r from https://github.com/posativ/isso.git but u can't get them directly so I created a project for the js files, u could see the original files "embed.js", "count.js". 
They r originally generated from the original project but not minified enough, so I minified them again!

**Comment by SnorlaxYum** [comment]
@pvnr0082t Sorry I forgot to commit the unminified ones. Now they have been committed.

File: ajax/libs/isso/package.json [hunk 2]
```diff
@@ -18,4 +18,17 @@
     "type": "git",
     "url": "https://github.com/posativ/isso.git"
   }
```
Comment (PeterDaveHello) [inline ajax/libs/isso/package.json]: missing comma here

File: ajax/libs/isso/package.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    "url": "https://github.com/posativ/"
+  },
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/posativ/isso.git"
+  }
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/SnorlaxYum/isso-js.git",
```
Comment (PeterDaveHello) [inline ajax/libs/isso/package.json]: Can we just use an official repo here?

Comment (SnorlaxYum) [inline ajax/libs/isso/package.json]: U couldn't get the js files from the official repo directly. That's why I created another repo

**Comment by SnorlaxYum** [comment]
@PeterDaveHello Just changed the URL. Now it uses my repo https://github.com/SnorlaxYum/isso-js .

**Comment by PeterDaveHello** [comment]
@SnorlaxYum We only accept official repo, should not use your own.

**Comment by SnorlaxYum** [comment]
Then I'll @posativ plz accept the merge request~

File: ajax/libs/isso/package.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/posativ/isso"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/posativ/isso.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
```
Comment (x09326) [inline ajax/libs/isso/package.json]: @SnorlaxYum 
I think you need to modify the `basePath`. There is no `dist` folder.

Comment (SnorlaxYum) [inline ajax/libs/isso/package.json]: As I said in that message, It will work once the Pull Request posativ/isso#289 is merged

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Comment by sufuf3** [comment]
@PeterDaveHello I think this PR can't host.
The repo is a lightweight commenting server written in Python and JavaScript. And it's not the JavaScript or CSS library which can use in front-end.
Also, the author seems didn't want to add minify files. (posativ/isso#289)
I think maybe we can close this PR.

**Comment by PeterDaveHello** [comment]
Close deu to no response.

**Pull request closed without merging**